### PR TITLE
Bring the ability to use influxdb (#2388)

### PR DESCRIPTION
**Pull request by evrardjp**
RPC-Maas is able to push metrics to influx, we should expose
this facility to our users, which includes our gates.

On top of it, we should bring the facility to deploy influx, if
need be.

RPC-Maas is able to push metrics to influx, we should expose
this facility to our users, which includes our gates.

Issue: [LA-314](https://rpc-openstack.atlassian.net/browse/LA-314)
Manual cherry-pick from eae30c4f74ad471cccfd6f17217dc6eeb0aa347b

Issue: [mitaka-13](https://rpc-openstack.atlassian.net/browse/mitaka-13)

### CODE REVIEW

**Comment by evrardjp** [comment]
This needs new maas to be merged into this branch.

**Comment by evrardjp** [comment]
``+ openstack-ansible --forks 4 maas-get.yml
Variable files: "-e @/etc/openstack_deploy/user_conf_files.yml -e @/etc/openstack_deploy/user_osa_aio_variables.yml -e @/etc/openstack_deploy/user_osa_secrets.yml -e @/etc/openstack_deploy/user_osa_variables_defaults.yml -e @/etc/openstack_deploy/user_osa_variables_overrides.yml -e @/etc/openstack_deploy/user_rpco_secrets.yml -e @/etc/openstack_deploy/user_rpco_variables_defaults.yml -e @/etc/openstack_deploy/user_rpco_variables_overrides.yml -e @/etc/openstack_deploy/user_zzz_gating_variables.yml "
ERROR: the playbook: maas-get.yml could not be found``

I will fix that.

**Comment by evrardjp** [comment]
depends on https://github.com/rcbops/rpc-openstack/pull/2345 that will not get fixed until a team decides to bypass the majorupgrade jobs, or, alternatively, fix them.

**Comment by cloudnull** [comment]
the major upgrade jobs are no longer required because the code path at this time is unmaintained and a blocker for progress. Please review #2345 when at all possible so we can get this done.

**Comment by cloudnull** [comment]
Updated to resolve merge conflicts

File: scripts/functions.sh [hunk 1]
```diff
@@ -1,4 +1,52 @@
...
+
+## Vars ----------------------------------------------------------------------
+
+# Gating
+export BUILD_TAG=${BUILD_TAG:-}
+export INFLUX_IP=${INFLUX_IP:-}
+export INFLUX_PORT=${INFLUX_IP:-"8086"}
+
+# Other
+export ADMIN_PASSWORD=${ADMIN_PASSWORD:-"secrete"}
```
Comment (hughsaunders) [inline scripts/functions.sh]: I don't like this, should be user secrets? Also seems to only be used for kibana so should labeled as such. However I realise this is a backport, so will let it go.

**Comment by evrardjp** [comment]
@cloudnull could you vote/merge?

File: scripts/functions.sh (hunk 1 above)
Comment (odyssey4me) [inline scripts/functions.sh]: Yeah, this is a bad habit holdover from bygone days.

**Pull request merged**